src/ui/atoms/text/Text.tsx

```diff
@@ -0,0 +1,53 @@
+import classNames from 'classnames'
+import React from 'react'
+import type { DeepReadonly } from '../../../superTypes'
+import './text.scss'
+
+export type TTextProps = DeepReadonly<{
+  /**
+   * Prop used to switch out what's being ultimately rendered. By default, a `span` component.
```

Comment:
Based on the @ccamel [comment](https://github.com/okp4/ui/pull/81#discussion_r843181124), maybe we need to remove the defaults notions here.